aspnet/signalr/overview/getting-started/tutorial-high-frequency-realtime-with-signalr.md

```diff
@@ -1,222 +1,276 @@
+
+## Add the client
+
+Next, you'll add the HTML client.
+
+1. In **Solution Explorer**, right-click the project and select **Add** > **HTML Page**.
+
+1. Name the page **Default** and select **OK**.
 
-1. Next, we'll add the client. In **Solution Explorer**, right-click the project, then click **Add | New Item**. In the **Add New Item** dialog, select **Html Page**. Name the page **Default.html** and click **Add**.
-2. In **Solution Explorer**, right-click the page you just created and click **Set as Start Page**.
-3. Replace the default code in the HTML page with the following code snippet.
+1. In **Solution Explorer**, right-click *Default.html* and select **Set as Start Page**.
 
-    > [!NOTE]
-    > Verify that the script references below match the packages added to your project in the Scripts folder.
+1. Replace the default code in the *Default.html* file with this code:
 
-    [!code-html[Main](tutorial-high-frequency-realtime-with-signalr/samples/sample3.html)]
+    [!code-html[Main](tutorial-high-frequency-realtime-with-signalr/samples/sample3.html?highlight=14-16)]
 
-    The above HTML and JavaScript code creates a red Div called Shape, enables the shape's dragging behavior using the jQuery library, and uses the shape's `drag` event to send the shape's position to the server.
-4. Start the application by pressing F5. Copy the page's URL, and paste it into a second browser window. Drag the shape in one of the browser windows; the shape in the other browser window should move.
+1. In **Solution Explorer**, expand **Scripts**.
 
-    ![The application window](tutorial-high-frequency-realtime-with-signalr/_static/image5.png)
+    Script libraries for jQuery and SignalR are visible in the project.
 
-<a id="clientloop"></a>
+    > [!IMPORTANT]
+    > The package manager will install a later version of the SignalR scripts.
+
+1. Update the script references in the code block to correspond to the versions of the script files in the project.
+
+This HTML and JavaScript code creates a red `div` called `shape`. It enables the shape's dragging behavior using the jQuery library and uses the `drag` event to send the shape's position to the server.
+
+## Run the app
+
+You can run the app to see it work. When you drag the shape around a browser window, the shape moves in the other browsers too.
+
+1. In the toolbar, turn on **Script Debugging** and then select the play button to run the application in Debug mode.
+
+    ![Screenshot of user turning on debugging mode and selecting play.](tutorial-high-frequency-realtime-with-signalr/_static/image3.png)
+
+    A browser window will open with the red shape in the upper-right corner.
+
+1. Copy the page's URL.
+
+1. Open another browser and paste the URL into the address bar.
+
+1. Drag the shape in one of the browser windows. The shape in the other browser window will follow.
+
+    ![GIF of the app in action](tutorial-high-frequency-realtime-with-signalr/_static/move-shape-demo.gif)
```

Comment:
Adding new images isn't part of the work planned for the MVC/content model updates. We generally try to minimize images, as they are time-consuming to keep updated. When working on an existing article, we keep only images that are essential to understanding the instructions. I appreciate the intent here, but I don't believe an animated gif is essential here to understand the app, and if added it becomes an additional item that must be kept updated.